django_project/innovation_module/idea.py

```diff
@@ -101,6 +98,13 @@ def block_idea(idea_id):
 def get_ideas(user, filter_user):
     
     objs = Pomysl.objects
+    user_obj = models.Uzytkownik.objects.get(user_id=user.id)
+    
+    normal_user = models.ZwyklyUzytkownik.objects.filter(uzytkownik_id=user.id)
+    print(normal_user.exists())
+    if normal_user.exists():
+        objs=models.Pomysl.objects.exclude(status_pomyslu='Zablokowany')
```

Comment:
testowałeś to dla zwykłego użytkownika? wydaje mi się że trzeba najpierw pobrać status pomysłu 'Zablokowany' a potem obiekt tego statusu dać do exclude

Reply:
i trzeba media dodać do .gitignore bo załączniki zaczęły się wrzucać XD

Reply:
testowałem, działa
a memy na szczęście względnie poprawne politycznie i słabe, bo tylko takie miałem na linuxie xd